packages/browser/src/shims/Replay.ts

```diff
@@ -0,0 +1,46 @@
+import type { Integration } from '@sentry/types';
+
+/**
+ * This is a shim for the Replay integration.
+ * It is needed in order for the CDN bundles to continue working when users add/remove replay
+ * from it, without changing their config. This is necessary for the loader mechanism.
+ */
+export class ReplayShim implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'Replay';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = ReplayShim.id;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(_options: any) {
+    // eslint-disable-next-line no-console
+    console.error('You are using new Replay() even though this bundle does not include the replay integration.');
```

Comment:
l: I'm wondering if we should actually log this out to everyone, given that people might actually do this on purpose. WDYT should we use the logger and gate it behind our debug flag? If you already discussed this, feel free to ignore :)

Reply:
Yeah, it's a good question. Do we want to "support" this being set when disabled? Not sure, IMHO it is more likely to be an accident than on purpose, but hard to say...?

Reply:
I'll leaf this as a console error for now, we can always loosen this up later if needed!